From bddd98fbabb5e32fdd731120c39c0549180e7521 Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Tue, 24 Oct 2023 09:22:58 +0200
Subject: [PATCH] remove long description

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 6c3aad9..dbf6b55 100644
--- a/setup.py
+++ b/setup.py
@@ -7,7 +7,7 @@
     description="Core Emmet Library",
     author="The Materials Project",
     author_email="feedback@materialsproject.org",
-    long_description=open("../README.md").read(),  # noqa: SIM115
+    long_description="",  # noqa: SIM115
     long_description_content_type="text/markdown",
     url="https://github.com/materialsproject/emmet",
     packages=find_namespace_packages(include=["emmet.*"]),
