From aed3753206a111e41951f78020f693e2ff9ac474 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Wed, 18 Jan 2023 10:08:42 -0700
Subject: [PATCH] Update setup.py

---
 setup.py | 11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/setup.py b/setup.py
index 484248d..9bcba1d 100644
--- a/setup.py
+++ b/setup.py
@@ -8,14 +8,7 @@
 
 setup(
     name="emmet-core",
-    use_scm_version={
-        "root": ".",
-        "relative_to": __file__,
-        "write_to": "emmet/core/_version.py",
-        "write_to_template": '__version__ = "{version}"',
-        "fallback_version": fallback_version,
-        "search_parent_directories": True,
-    },
+    version=fallback_version,
     setup_requires=["setuptools_scm>=6,<8"],
     description="Core Emmet Library",
     author="The Materials Project",
@@ -30,7 +23,7 @@
         "pydantic==1.10.2",
         "pybtex~=0.24",
         "typing-extensions>=3.7,<5.0",
-        "spglib==2.0.1",
+        "spglib>=2.0.1",
     ],
     extras_require={
         "all": ["robocrys>=0.2.7", "pymatgen-analysis-diffusion>=2022.1.15", "pymatgen-analysis-alloys>=0.0.3"],
