From d3486e0f19e1c975f7d795c9033d31fcee8c2d2b Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Fri, 21 Oct 2022 09:42:53 -0600
Subject: [PATCH] Update _version.py

---
 emmet-core/_version.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/emmet-core/_version.py b/emmet-core/_version.py
index ee2ebcf94..53b39ad36 100644
--- a/emmet-core/_version.py
+++ b/emmet-core/_version.py
@@ -1 +1 @@
-__version__ = "0.38.0"
\ No newline at end of file
+__version__ = "0.38.1"
\ No newline at end of file
