From f45c8d5cbb5afa327208297fbe138426f0cd2aeb Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan.janssen@outlook.com>
Date: Fri, 21 Oct 2022 09:21:58 -0600
Subject: [PATCH] patch version

---
 emmet-core/_version.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/emmet-core/_version.py b/emmet-core/_version.py
index 2d93b0cd2..978180b4c 100644
--- a/emmet-core/_version.py
+++ b/emmet-core/_version.py
@@ -1 +1 @@
-__version__ = "0.38.0"
+__version__ = "0.38.1"
